documentation/release_notes.rst

```diff
@@ -34,22 +34,22 @@ Fixed Issues
 - Fixed several issues in the ``sort``, ``stable_sort``, ``sort_by_key`` and ``stable_sort_by_key`` algorithms that:
 
    * Allows the use of non-trivially-copyable comparators.
-   * Eliminates duplicate kernel names
+   * Eliminates duplicate kernel names.
    * Resolves incorrect results on devices with sub-group sizes smaller than four.
    * Resolved synchronization errors that were seen on Intel® Arc™ B-series GPU devices.
 
 Known Issues and Limitations
 ----------------------------
 New in This Release
 ^^^^^^^^^^^^^^^^^^^
-- Incorrect results may be observed when calling ``sort`` with a device policy on Intel® Arc™ Graphics 140V with data
+- Incorrect results may be observed when calling ``sort`` with a device policy on Intel® Arc™ graphics 140V with data
   sizes of 4-8 million elements.
 - ``sort``, ``stable_sort``, ``sort_by_key`` and ``stable_sort_by_key`` algorithms fail to compile
   when using Clang 17 and earlier versions, as well as compilers based on these versions,
-  such as Intel(R) oneAPI DPC++/C++ Compiler 2023.2.0.
+  such as Intel® oneAPI DPC++/C++ Compiler 2023.2.0.
```

Comment:
Similarly here we can just omit the trademark.

Reply:
Not correct. See above guidance.